### DISCUSSION TITLE
Difference between lossless and hi-res option on tidal

### DISCUSSION THREAD

**Discussion by dgpapagian** [music-assistant/support#2377] [Q&A] [answered]
title.

**Comment by OzGav** [comment]
Did you try Google? https://www.google.com/search?q=tidal+difference+lossless+and+hi+res&oq=tidal+difference+lossless+and+hi+res

**Reply by OzGav** [reply, marked as answer]
Tidal config has been updated and info added
<img width="506" alt="image" src="https://github.com/user-attachments/assets/154b8569-f7c7-42d2-b0b4-3c6c7890b726">
